src/test/java/org/jvnet/hudson/test/RealJenkinsRuleSyntheticPluginTest.java

```diff
@@ -58,4 +60,15 @@ private static void _smokes(JenkinsRule r) throws Throwable {
         });
     }
 
+    @Test public void dynamicLoad() throws Throwable {
+        var file = temp.newFile("synthetic-stuff.jpi");
+        rr.then(r -> {
+            RealJenkinsRule.createSyntheticPlugin(Stuff.class.getPackage())
+                    .shortName("synthetic-stuff")
+                    .writeTo(file, Jenkins.getVersion().toString());
+            r.jenkins.pluginManager.dynamicLoad(file);
+            assertThat(r.createWebClient().goTo("stuff", "text/plain").getWebResponse().getContentAsString(),
+            is(Jenkins.get().getLegacyInstanceId()));
```

Comment:
```suggestion
                is(Jenkins.get().getLegacyInstanceId()));
```
(enabling Spotless would be a good idea, the flux of PRs in this repo is just pretty high)

Reply:
Indented in [056e148](https://github.com/jenkinsci/jenkins-test-harness/pull/920/commits/056e148972f94c931cf409cfceb5bba1a9d77c03)